src/components/auth/AuthLayout.tsx

```diff
@@ -1,52 +1,53 @@
-import { cloneElement, Component } from 'react';
-import PropTypes from 'prop-types';
+import {
+  cloneElement,
+  Component,
+  MouseEventHandler,
+  ReactElement,
+} from 'react';
 import { observer } from 'mobx-react';
 import { TitleBar } from 'electron-react-titlebar/renderer';
-
-import { injectIntl } from 'react-intl';
+import { injectIntl, WrappedComponentProps } from 'react-intl';
 import { mdiFlash } from '@mdi/js';
+import { GlobalError } from 'src/@types/ferdium-components.types';
```

Comment:
Please use relative paths - that's the standard we have been using till date

Reply:
will follow relative path format